src/components/settings/SettingsLMSTab/LMSConfigs/Degreed2Config.jsx

```diff
@@ -150,6 +164,27 @@ const Degreed2Config = ({ enterpriseCustomerUuid, onClick, existingData }) => {
             </Form.Control.Feedback>
           )}
         </Form.Group>
+        <Form.Group className="my-4">
+          <Form.Control
+            type="text"
+            maxLength={255}
+            isInvalid={!fetchUrlValid}
+            onChange={(e) => {
+              setEdited(true);
+              validateField('Degreed Token Fetch Base Url', e.target.value);
+            }}
+            floatingLabel="Degreed Token Fetch Base Url"
+            defaultValue={existingData.degreedFetchUrl}
+          />
+          <Form.Text>
+            Optional: If provided, will be used as base url instead of Degreed Base URL to fetch tokens.
```

Comment:
Did this text come from UX? Idk why but reading "will be used as base url..." made me think this field would replace the other base url field. It could say "be used as the url to fetch API tokens..." or something but I could also very well be crazy. What do you think?

Reply:
No it didn't come from UX, just from the model description. I'll change it to what you suggested!